spec/services/inherited_proofing/va/service_spec.rb

```diff
@@ -15,6 +15,12 @@
 
   let(:auth_code) {}
   let(:private_key) { private_key_from_store_or(file_name: 'va_ip.key') }
+  let(:payload) { { inherited_proofing_auth: auth_code, exp: 1.day.from_now.to_i } }
+  let(:jwt_token) { JWT.encode(payload, private_key, 'RS256') }
+  let(:request_uri) {
+    "#{ URI(InheritedProofing::Va::Service::BASE_URI) }/inherited_proofing/user_attributes"
```

Comment:
this interpolation is a bit redundant, `#{}` will call `#to_s` and so `URI(...).to_s` is the same as just interpolating in the constant?
```suggestion
    "#{InheritedProofing::Va::Service::BASE_URI}/inherited_proofing/user_attributes"
```

Reply:
Good catch! I will clean that up